### ISSUE TITLE
Proposal for supporting HTML `lang` attribute

### ISSUE THREAD

**Issue by LoganVS** [jantimon/html-webpack-plugin#562]
In pursuit of the notion that the web should be an accessible place for everyone, I am interested in adding support for the html `lang` attribute.  `lang` is import in web accessibility, particularly by telling a screenreader what language a page is in. It is therefore necessary for complying with the minimum set of web accessibility guidelines [(WCAG 20)](https://www.w3.org/WAI/WCAG20/quickref/?currentsidebar=%23col_customize&levels=aa%2Caaa&showtechniques=311).

I would like to add a `lang` or `language` option to the html-webpack-plugin options, which would then set the `lang` attribute in the html tag.  Another possible approach would be to accept an array of key value pairs that could then be set on html, body, etc; however, I don't necessarily see the demand or necessity for such a feature (perhaps you do) so I feel like that would be overkill.

Please tell me what you think of this proposal.  If you like it I would be happy to contribute to making it happen!

**Comment by jantimon** [comment]
This can already be done in your template but might also be a good idea for 

https://github.com/jaketrent/html-webpack-template

**Comment by LoganVS** [comment]
Hey @jantimon, thanks for your reply.  I'm currently using a template- but only so I can include `lang`.  I thought it was a shame that this was the only reason I needed a template.  

I just checked out out html-webpack-template, and looks like there is already a [PR](https://github.com/jaketrent/html-webpack-template/pull/35) open.  I'll try and finish that so others can use this feature.

Can you speak more as to your ideas behind accessibility linting?


Thanks so much.

**Comment by jantimon** [comment]
Well there were a lot of ticket saying I am fine with the default template all I need would be feature X.

We decided to not implement most of them to keep this plugin as simple as possible.

There are plugins like https://github.com/yargalot/AccessSniff which analyse the html code and tell you to add `title` `lang` `alt` and similar

**Comment by LoganVS** [comment]
Ok sure, I understand.

Let me look at html-webpack-template and try and finish implementing this feature there.

Thanks for taking the time!
